examples/glTFViewer/glTFViewer.cpp

```diff
@@ -496,6 +520,7 @@ namespace {
             material.uniformBuffer.SetSubData(0,
                     sizeof(u_transform_block) / sizeof(uint32_t),
                     reinterpret_cast<const uint32_t*>(&transforms));
+            cmd.BeginRenderPass(material.renderpass, material.framebuffer);
```

Comment:
This is missing an EndRenderpass, can you add validation for currentRenderpass == nullptr at the end of a CommandBuffeBuilder::ValidateGetResult(); (and ideally a test for it somewhere in the validation tests.)